docs/source/basics.rst

```diff
@@ -388,6 +404,14 @@ You can achieve the oft-used `control-off` gate (flip the target qubit
 
    prog = Program(X(0), X(1).controlled(0), X(0))
 
+The gate ``FORKED RX(pi/2, pi) 0 1`` may be produced by
+
+.. code:: python
+
+   prog = Program(RX(np.pi/2, 1).forked(0, np.pi))
```

Comment:
Does that `np.pi` need to wrapped in list?

Reply:
Nice catch. 👀